exporter/prometheusexporter/accumulator.go

```diff
@@ -204,6 +209,12 @@ func (a *lastValueAccumulator) accumulateSum(metric pmetric.Metric, il pcommon.I
 			continue
 		}
 
+		// Delta-to-Cumulative
+		if doubleSum.AggregationTemporality() == pmetric.MetricAggregationTemporalityDelta {
```

Comment:
I believe we only want to perform the accumulation inside this branch when the StartTimestamp of the arriving point equals the last-observed Timestamp of the stored point.

Reply:
updated accordingly.